src/builds/node/build-parser.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export interface IBuildLogParser {
+    /** starts the parsing */
+    parse(): Promise<IParsedBuildLog>
```

Comment:
These are JSDoc documentation entries, similar to JavaDoc. It permits having those comments be displayed when hovering over the fields.